automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -0,0 +1,150 @@
+import os
+import sys
+import time
+
+from tools import reboot
+from automation_tools import (
+    set_yum_debug_level,
+    setup_satellite_firewall,
+    subscribe
+)
+from automation_tools.satellite6.hammer import hammer, set_hammer_config
+from automation_tools.repository import enable_repos, disable_repos
+from automation_tools.utils import distro_info, update_packages
+from fabric.api import env, execute, put, run
+from tasks import create_rhevm_instance, delete_rhevm_instance
```

Comment:
Prefer absolute imports, or use explicit relative imports for tasks and tools...